src/SfxWeb/src/app/shared/component/dual-date-picker/dual-date-picker.component.ts

```diff
@@ -49,13 +49,12 @@ export class DualDatePickerComponent implements OnInit, OnChanges {
   onDateSelection(date: NgbDate) {
     if (!this.fromDate && !this.toDate) {
       this.fromDate = date;
-    } else if (this.fromDate && !this.toDate && date.after(this.fromDate)) {
+    } else if (this.fromDate && !this.toDate && !date.before(this.fromDate)) {
```

Comment:
why this change?

Reply:
before the change, the calender was not allowing to choose the same start and end dates, which should be possible to do. I should be able to filter choosing a particular date

Reply:
sounds good, was not aware this was an issue.